sdk/cosmosdb/cosmos/samples-dev/ServerSideScripts.ts

```diff
@@ -65,7 +61,7 @@ const sprocDefinition = {
 
     // To replace the document, first issue a query to find it and then call replace.
     function tryReplace(doc: Item, cback: any) {
-      retrieveDoc(doc, null, function(retrievedDocs: Resource[]) {
+      retrieveDoc(doc, '', function(retrievedDocs: Resource[]) {
```

Comment:
If continuation isn't mandatory, you can make it optional in the signature of `retrieveDoc` instead of passing in an empty string.

Reply:
fixed